CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Unreleased
 
 ### Changed
 * Updated terra-form-select dependency to ^6.0.0
+* Removed the locale configuration in favor of using the global `TERRA_AGGREGATED_LOCALES` variable provided by terra-toolkit v5.21.0.
```

Comment:
This guy needs to be moved into un-released changes because I released a new version yesterday

Reply:
Moved to un-released, thanks for catching that.